.github/workflows/ci.yml

```diff
@@ -9,6 +9,12 @@ on:
 
 jobs:
   build:
+    env:
+      MAVEN_OPTS: " --add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED \
+      --add-exports jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED \
+      --add-exports jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED \
+      --add-exports jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED \
+      --add-exports jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED "
```

Comment:
Reicht das nicht in der `jvm.config`?

Reply:
Das weiß ich eben nicht. Man könnte es mal ausprobieren. Nimm's mal raus und schau, was die CI sagt :wink:

Reply:
Geht auch ohne :smile: